src/main/kotlin/com/coxautodev/graphql/tools/SchemaClassScanner.kt

```diff
@@ -142,7 +145,7 @@ internal class SchemaClassScanner(initialDictionary: BiMap<String, Class<*>>, al
         // Ensure all scalar definitions have implementations and add the definition to those.
         val scalars = scalarDefinitions.filter { !ScalarInfo.STANDARD_SCALAR_DEFINITIONS.containsKey(it.name) }.map { definition ->
             val provided = scalars[definition.name] ?: throw SchemaClassScannerError("Expected a user-defined GraphQL scalar type with name '${definition.name}' but found none!")
-            GraphQLScalarType(provided.name, SchemaParser.getDocumentation(definition) ?: provided.description, provided.coercing, definition)
+            GraphQLScalarType(provided.name, SchemaParser.getDocumentation(definition) ?: provided.description, provided.coercing, ArrayList<GraphQLDirective>(), definition)
```

Comment:
Added an emptyList. The new builder part of `GraphQLScalarType` did not allow `Coercing coercing` to be set. Have used the only constructor that allows me to pass the args required. With this in mind I have had to pass an empty list  `ArrayList<GraphQLDirective>()`, as it cannot be null.

Reply:
Would you mind changing this to `listOf()`?